types/moment-timezone/index.d.ts

```diff
@@ -55,7 +54,15 @@ declare module "moment" {
         names(): string[];
         guess(ignoreCache?: boolean): string;
 
-        setDefault(timezone?: string): MomentTimezone;
+        setDefault(timezone: string): MomentTimezone;
+
+        pack(data: object): string;
+
+        packBase60(x: any, y: number): any;
+
+        unpack(data: string): object;
```

Comment:
Same comment, the `object` returned is identical to the `Zone` object as stated in the documentation.